src/components/layouts/MainLayout.tsx

```diff
@@ -1,13 +1,18 @@
 import type { ReactNode } from "react";
 
+import Footer from "@/components/Footer";
+
 interface Props {
   children: ReactNode | ReactNode[];
 }
 
 export default function MainLayout({ children }: Props) {
   return (
-    <main className="flex min-h-screen flex-col items-center justify-between p-24">
-      {children}
-    </main>
+    <>
+      <main className="flex min-h-screen flex-col items-center justify-between p-24">
+        {children}
+      </main>
+      <Footer></Footer>
```

Comment:
It would be better to use one tag top open and close if we aren't putting anything in between:
```<Footer />```